apps/daimo-mobile/.maestro/config.yaml

```diff
@@ -0,0 +1,8 @@
+# Maestro Cloud configuration
+baselineBranch: master
+notifications:
+  email:
+    enabled: true
+    onSuccess: true
+    recipients:
+      - alerts-aaaak7okwrieedyz372hhd57ue@daimo-workspace.slack.com
```

Comment:
this token is public now.. does this let people do anything other than post to our  slack?

(email on GH = crawlers will likely pick it up & we'll get spam)

recommend using env var